src/config/settings.js

```diff
@@ -0,0 +1,201 @@
+/*
+| -----------------------------------------------------------------------------
+| Unified InstantSearch E-commerce by Algolia             http://alg.li/unified
+| -----------------------------------------------------------------------------
+|
+| Welcome to the Unified InstantSearch E-commerce configuration file.
+| This is where you can customize Unified InstantSearch E-commerce specifically
+| for your project.
+|
+| Unless mandatory, if you don't use a setting, feel free to comment it out.
```

Comment:
It might not be clear enough which settings are mandatory? I had to carefully read the "Base configuration" to understand that you were referring to those.

Perhaps we could say the following?

> Unless settings from "Base configuration" which are mandatory, feel free to comment them out.

Reply:
The `keyboardShortcuts` parameter isn't mandatory, but I don't see where else it belongs.

Reply:
I will resolve it so we can merge this PR, and we can think of a better way to phrase this in another one.